src/components/modals/DarModal.js

```diff
@@ -32,37 +42,52 @@ const processResearchTypes = (researchTypes) => {
   return researchStatements;
 };
 
+const requiresManualReview = (darDetails) => {
+  return darDetails.illegalBehavior ||
+    darDetails.poa ||
+    darDetails.psychiatricTraits ||
+    darDetails.sexualDiseases ||
+    darDetails.stigmatizedDiseases ||
+    darDetails.vulnerablePopulation;
+};
+
 const DarModal = (props) => {
   //NOTE: Modal should be simple (raw information should be passed in as props) in order to ensure plug and play use
-  const {showModal, closeModal, darDetails} = props;
+  const {showModal, closeModal, darDetails, researcher} = props;
 
   return h(Modal, {
     isOpen: showModal,
     onRequestClose: closeModal,
     shouldCloseOnOverlayClick: true,
     style: {
-      content: Styles.MODAL.CONTENT
+      content: {
+        ...Styles.MODAL.CONTENT,
+        ...{boxShadow: '3px 3px 0 #cccccc', borderRadius: "10px"}
```

Comment:
Instead of using a box-shadow, change the overlay to make the foreground a bit darker.

```
style: {
      content: {
        ...Styles.MODAL.CONTENT,
      },
      overlay: {
        backgroundColor: 'rgba(0, 0, 0, 0.5)'
      }
    }
 ```

Reply:
Thank you! 👍🏽